src/test/java/com/qa/restassured/TestRestAssured.java

```diff
@@ -0,0 +1,17 @@
+package com.qa.restassured;
+import static io.restassured.RestAssured.*;
+import static io.restassured.matcher.RestAssuredMatchers.*;
+import static org.hamcrest.Matchers.*;
+
+import org.testng.annotations.Test;
+
+public class TestRestAssured {
+
+    @Test
+    public void test(){
```

Comment:
Why do you want to name a test method just test? what does it do? what kind of test is it?
Please focus on naming better, this kind of habit might be very dangerous , means beginner level of coding

Reply:
Modify